assets/Assets.php

```diff
@@ -9,7 +9,11 @@ class Assets extends AssetBundle
 {
     public $sourcePath = '@socialshare/resources';
 
+    public $js = [
+        'js/humhub.socialshare.ShareLink.js'
+    ];
+
     public $publishOptions = [
-        'forceCopy' => 'false',
+        'forceCopy' => 'false'
```

Comment:
if `false` is required here, then it should be used as boolean type and not string `'false'`, otherwise the string works as boolean `true` I think.

Reply:
I didn't think it really matter but if preferred I changed it, otherwise I would have removed it completely, didn't know if @luke- would prefer it to stay in or not. 🤔